src/Core/src/Platform/iOS/ContentView.cs

```diff
@@ -70,7 +99,9 @@ public override void SetNeedsLayout()
 			Superview?.SetNeedsLayout();
 		}
 
+		[Obsolete("Use View instead. This member causes memory leaks.")]
 		internal Func<double, double, Size>? CrossPlatformMeasure { get; set; }
+		[Obsolete("Use View instead. This member causes memory leaks.")]
```

Comment:
@jonathanpeppers @hartez since this is internal, should we maybe change the API so we can use a static delegate and rather pass around some sort of state - I am looking at the code for scroll view and the fact that it seems to sort of make some decisions with these 2 delegates.

Reply:
I left these here for `ScrollView` to be fixed later.

How do we enforce the delegate will always be static?